src/app.js

```diff
@@ -429,13 +430,23 @@ var run = function () {
   var cachedRemoteFiles = {}
 
   function handleImportCall (url, cb) {
-    var githubMatch
+    var match
     if (editor.hasFile(url)) {
       cb(null, editor.getFile(url))
     } else if (url in cachedRemoteFiles) {
       cb(null, cachedRemoteFiles[url])
-    } else if ((githubMatch = /^(https?:\/\/)?(www.)?github.com\/([^/]*\/[^/]*)\/(.*)/.exec(url))) {
-      handleGithubCall(githubMatch[3], githubMatch[4], function (err, content) {
+    } else if ((match = /^(https?:\/\/)?(www.)?github.com\/([^/]*\/[^/]*)\/(.*)/.exec(url))) {
+      handleGithubCall(match[3], match[4], function (err, content) {
+        if (err) {
+          cb('Unable to import "' + url + '": ' + err)
+          return
+        }
+
+        cachedRemoteFiles[url] = content
+        cb(null, content)
+      })
+    } else if ((match = /^bzzr:\/\/?([0-9a-fA-F]{64}$)/.exec(url))) {
```

Comment:
I think we should also support `bzz://` because that supports directory structures.